packages/config/config/devices/0x015f/mh-3901z.json

```
@@ -0,0 +1,129 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH3901-Z",
+	"description": "Boiler Thermostat",
+	"devices": [
+		{
+			"productType": "0x3910",
+			"productId": "0x5102"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Temperature reporting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "Difference reporting only",
+					"value": 1
+				},
+				{
+					"label": "Interval reporting only",
+					"value": 2
+				},
+				{
+					"label": "Difference and interval reporting",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Temperature difference setting",
+			"description": "Based on 0.1 °C unit: n * 0.1 °C = 0.5 °C",
+			"unit": "0.1 °C",
+			"valueSize": 2,
+			"minValue": 3,
+			"maxValue": 995,
+			"defaultValue": 5,
+			"unsigned": true
+		},
+		{
+			"#": "3",
+			"label": "Report interval time",
+			"description": "Based in 1s unit. Suggested > 30s",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 32767,
+			"defaultValue": 60,
+			"unsigned": true
+		},
+		{
+			"#": "5",
+			"label": "Association 2 SET command retransmission times",
+			"description": "0: not resend, 1-127: resend times",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 3
+		},
+		{
+			"#": "7",
+			"label": "Set the number of units for the heating period (menu P07)",
+			"description": "Total length of a heating cycle",
```

Labels should be short, additional information goes in the description.
```suggestion
			"label": "Heating Cycle Duration",
			"description": "This can also be configured in menu P07",
```

---

Please check the last commit.